ROADMAP.md

```diff
@@ -0,0 +1,89 @@
+# Roadmap Native AuthN/AuthZ
```

Comment:
Should we call this `SECURITY_ROADMAP.md`?

Reply:
We may want to turn this list into checkboxes and check items off as we go.

Reply:
Lets checkboxify this after we have flushed these items out more, it might make sense to create individual issues that can be assigned to folks and further granularized.  I do really like the idea of being able to see the list of items start completing

Reply:
I ended up going with Identity Roadmap instead of Security.